packages/cyberstorm/src/components/Providers.tsx

```diff
@@ -1,15 +1,18 @@
 "use client";
 import * as RadixTooltip from "@radix-ui/react-tooltip";
 import { ReactNode } from "react";
+import { ToastProvider } from "./Toast/ToastContext";
 
 interface CyberstormProvidersProps {
   children: ReactNode | ReactNode[];
 }
 
 export function CyberstormProviders(props: CyberstormProvidersProps) {
   return (
-    <RadixTooltip.Provider delayDuration={80}>
-      {props.children}
-    </RadixTooltip.Provider>
+    <ToastProvider>
```

Comment:
Could be handy if `CyberstormProviders` could accept toast's duration and tooltip's delayDuration as props. Now we are hardcoded to use one values in all our projects.

Not a blocker.